### DISCUSSION TITLE
Refresh Discord Bot Profile Pictures

### DISCUSSION THREAD

**Discussion by MisterNormie** [EssentialsX/Essentials#4736] [Ideas and Feature Suggestions]
Hey; id like to suggest the ability to refresh the discord bot pictures when a player changes their minecraft skin. Either through a command or on a timer.

**Comment by Chew** [comment]
I'm not sure you understand how webhooks work. When it's sent, the icon can never be changed. If a player's skin is changed, it'll only affect new messages, it's impossible to change old ones. Unless I'm misinterpreting, this isn't possible.

**Reply by JRoy** [reply]
> I'm not sure you understand how webhooks work. When it's sent, the icon can never be changed. If a player's skin is changed, it'll only affect new messages, it's impossible to change old ones. Unless I'm misinterpreting, this isn't possible.

Discord caches based on URL's, I could prepend the texture key to force discord to update its cache

**Reply by MisterNormie** [reply]
> 

I was meaning new messages still using the old skin :)

**Reply by MisterNormie** [reply]
> 

Oooh if that is indeed possible that would be great. I changed my skin 2 weeks ago and its still showing the old one in the picutres on new messages so an ability to make discord fetch new icons would be sweet!
